src/testHelpers.ts

```diff
@@ -5,9 +5,12 @@ import { BigNumber, providers } from "ethers";
 import { Wallet } from "ethers/lib/ethers";
 import { NetworkId } from "src/constants";
 import { DecimalBigNumber } from "src/helpers/DecimalBigNumber/DecimalBigNumber";
+import { chains as allChains } from "src/hooks/wagmi";
 import { vi } from "vitest";
-import { allChains, Chain, chain as chain_, createClient, CreateClientConfig } from "wagmi";
+import { Chain, createClient, CreateClientConfig } from "wagmi";
 import * as WAGMI from "wagmi";
+import { hardhat } from "wagmi/chains";
```

Comment:
this was the MAJOR set of issues that actually needed to be fixed. Other changes in this pr were just fixing unstable tests.